packages/ui/src/components/InputContainer/styles.module.css

```diff
@@ -8,9 +8,10 @@
         border-radius: var(--go-ui-border-radius-md);
         background-color: var(--go-ui-color-element-background);
         padding: 0 var(--go-ui-spacing-sm);
-
+        
         .input {
-            padding: var(--go-ui-spacing-xs) 0;
+            padding-left:12px;
+            height: 24px;
```

Comment:
@frozenhelium Not sure if this is the way to set fixed height for inputs.
Also we should use design tokens.

Reply:
It was requested by the auditor